.prettierignore

```diff
@@ -18,10 +18,13 @@ AppXManifest.xml
 copyright-header.txt
 dist/
 docs/LICENSE.txt
+src/electron/resources/license_en.txt
+src/electron/resources/license.txt
 docs/NOTICE.html
 drop/
 extension/
 LICENSE
 new-appx.ps1
 test-results/
 yarn.lock
+LICENSE.txt

```

Comment:
I see a `LICENSE.txt` under the `docs` folder but not directly on the root folder. Is this addition to `.prettierignore` meant to ignore a file we didn't add on this PR?

Reply:
I should have deleted it, but I forgot